evaluate/repository.go

```diff
@@ -29,31 +29,31 @@ func Repository(logger *log.Logger, resultPath string, model evalmodel.Model, la
 
 	dataPath := filepath.Join(testDataPath, repositoryPath)
 
-	temporaryPath, err := os.MkdirTemp("", "eval-dev-quality")
```

Comment:
Not a workaround. It is a fix until we have a better solution. ALso. why is missing.

Reply:
And we are not resetting.

Reply:
Early merge as well.